sdk/clientcore/README.md

```diff
@@ -1,32 +1,32 @@
-# Azure Core shared library for Java
+# Client Core shared library for Java
 
-[![Build Documentation](https://img.shields.io/badge/documentation-published-blue.svg)](https://azure.github.io/azure-sdk-for-java)
+<!-- [![Build Documentation](https://img.shields.io/badge/documentation-published-blue.svg)](https://azure.github.io/azure-sdk-for-java) -->
 
-Azure Core provides shared primitives, abstractions, and helpers for modern Java Azure SDK client libraries. These libraries follow the [Azure SDK Design Guidelines for Java](https://azure.github.io/azure-sdk/java_introduction.html) and can be easily identified by package names starting with `io.clientcore` and module names starting with `azure-`, e.g. `io.clientcore.storage.blobs` would be found within the `/sdk/storage/azure-storage-blob` directory. A more complete list of client libraries using Azure Core can be found [here](https://azure.github.io/azure-sdk/releases/latest/#java-packages).
+Client Core provides shared primitives, abstractions, and helpers for modern SDK client libraries. These libraries
+<!-- follow the [SDK Design Guidelines for Java](https://azure.github.io/azure-sdk/java_introduction.html) and -->
+can be easily identified by package names starting with `io.clientcore` and module names starting with `core`,
+e.g. `io.clientcore.core` would be found within the `/sdk/clientcore/core` directory.
+<!-- A more complete list of client libraries using Core can be found [here](https://azure.github.io/azure-sdk/releases/latest/#java-packages). -->
 
-Azure Core allows client libraries to expose common functionality in a consistent fashion, so that once you learn how to use these APIs in one client library, you will know how to use them in other client libraries.
+Client Core allows client libraries to expose common functionality in a consistent fashion, so that once you learn how
+to use these APIs in one client library, you will know how to use them in other client libraries.
 
-The main shared concepts of Azure Core (and therefore all Azure client libraries using Azure Core) include:
+The main shared concepts of Client Core include:
 
 - Configuring service clients, e.g. configuring retries, logging, etc.
 - Accessing HTTP response details (`Response<T>`).
-- Calling long running operations (`Poller<T>`).
-- Paging and asynchronous streams (`PagedFlux<T>`).
+<!-- - Calling long-running operations (`Poller<T>`). -->
```

Comment:
What do you think about just removing these instead of commenting?

Reply:
Sounds good to me, I wasn't sure if we'd ever add anything related and was keeping them around just in case.